packages/components/atoms/f-button/src/components/Action.vue

```diff
@@ -0,0 +1,5 @@
+<template>
+    <button type="button">
```

Comment:
You probably want to set this dynamically? You could pass through an option to allow attributes to be passed through and then you could dynamically set them via a directive? Maybe make use of the prop on the button.vue component that exposes the $attrs already? Happy for this to be looked in another pr btw.

Reply:
@kevinrodrigues I _think_ I've got this working now 🤞